UKHomeOffice/nmsw-ui
# NMSW-453 Voyage pages - remove upload a file label and make H1 the label

# Ticket

NMSW=453

----

## AC

Refactor the voyage pages to remove the `Upload file` label and use the H1 as the label instead

----

## To test

- Run app
- Go to localhost:3000
- Sign in
- Click on 'Report a voyage'
- > The h1 should also function as a label with the supporting text above the input
- Compare the page to the updated page in the prototype (You may have to clear the prototype data to see the change)

----

## Notes
